config/routes.rb

```diff
@@ -14,9 +12,10 @@
 get "user/login"
 get "user/sign_up"
 get "employee/employee"
-get "laptop/laptop"
 get "employees/index"
 
+get 'assets/laptops', to: 'assets#laptops', as: 'laptops'
```

Comment:
Are you saying there is an error when you put the route as get 'asset-directory/laptops', to: 'assets#laptops', as: 'laptops'?